PostBoard.Api/Controllers/BirthdayController.cs

```diff
@@ -51,6 +60,14 @@ public IActionResult Create([FromBody] Birthday input)
     [Route("{id:int}")]
     public IActionResult Update([FromRoute] int id, [FromBody] Birthday input)
     {
+        var validator = new BirthdayValidator();
+        var result = validator.Validate(input);
```

Comment:
Rename `result` to `validationResult`, so we understand that it's not method result.